src/ui/sass/modules/_DirectAnswer.scss

```diff
@@ -81,7 +81,7 @@ $direct-answer-footer-height: var(--yxt-module-footer-height) !default;
   {
     display: flex;
     margin-right: calc(var(--yxt-base-spacing) / 2);
-    height: 18px;
```

Comment:
It looks like this is styling that has been there for 7 months. Is this a breaking change we want to happen? (It looks like this class was used in v1.3.0)

Reply:
we're actually using yxt-Results-titleIconWrapper in 1.3.0 right now, instead of this yxt-DirectAnswer-titleIconWrapper, so shouldn't be breaking!